interpreter/cling/lib/Interpreter/Interpreter.cpp

```diff
@@ -527,15 +527,17 @@ namespace cling {
   Interpreter::CompilationResult
   Interpreter::process(const std::string& input, Value* V /* = 0 */,
                        Transaction** T /* = 0 */) {
-    if (isRawInputEnabled() || !ShouldWrapInput(input))
-      return declare(input, T);
-
     CompilationOptions CO;
     CO.DeclarationExtraction = 1;
```

Comment:
I believe your patch changes the value of this option (and a few below) if `ShouldWrapInput`. Could you check, please?